src/Shortcodes/Shortcode.php

```diff
@@ -74,6 +75,7 @@ protected function addActions()
         protected function addShortcodes()
```

Comment:
is there a reason for using protected here ?

Reply:
Yes the `addShortcodes` method will not we used publicly, same as `addFilters` and `addActions`
those are called in `__construct()` for the `Shortcode` class